docs/content/latest/benchmark/sysbench.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Benchmark YSQL performance using Sysbench
+headerTitle: Sysbench
+linkTitle: Sysbench
+description: Benchmark YugabyteDB YSQL performance using Sysbench.
+headcontent: Benchmark YugabyteDB YSQL performance using Sysbench.
+menu:
+  latest:
+    identifier: sysbench-ysql
+    parent: benchmark
+    weight: 5
+aliases:
+  - /benchmark/sysbench/
+showAsideToc: true
+isTocNested: true
+---
+
+{{< note title="Note" >}}
+
+This is a fork of the sysbench utility with a few additions to better reflect
+the database's distributed nature.
```

Comment:
```suggestion
YugabyteDB's distributed nature.
```

Reply:
I went to the repo fork and don't see anything about how sysbench was modified to "better reflect YugabyteDB's distributed nature." That should be mentioned here and also in the repo.

Reply:
Done.